x-pack/plugins/apm/public/components/shared/charts/helper/get_alert_annotations.test.tsx

```diff
@@ -23,27 +23,27 @@ const theme = ({
 } as unknown) as EuiTheme;
 const alert: Alert = {
   'rule.id': ['apm.transaction_duration'],
-  'kibana.rac.alert.evaluation.value': [2057657.39],
+  'kibana.alert.evaluation.value': [2057657.39],
```

Comment:
Can we use constants from '@kbn/rule-data-utils` here instead of the hardcoded strings? So that we don't have to update these lines manually once something changes.

Reply:
@xcrzx I modified all the `kibana.alert.*` instances to use the constants. I haven't modified the others, because there's an enormous number of them. 😂  And most of those are unlikely to change. If you're okay with this for now, we can do the rest later...

Reply:
Yeah, I can imagine there ought to be bazillions of these hardcoded strings 🙂  For sure, no need to replace them all in this PR.